products/apache-camel.md

```diff
@@ -0,0 +1,218 @@
+    eol: 2022-12-31
+    lts: true
+    supportedJavaVersions: 8, 11
+    latest: "3.14.7"
+    latestReleaseDate: 2022-12-13
+
+-   releaseCycle: "3.13"
+    releaseDate: 2021-11-08
+    eol: 2021-11-08
+    lts: false
+    supportedJavaVersions: 8, 11
+    latest: "3.13.0"
+    latestReleaseDate: 2021-11-08
+
+-   releaseCycle: "3.12"
+    releaseDate: 2021-09-28
+    eol: 2021-09-28
+    lts: false
+    supportedJavaVersions: 8, 11
+    latest: "3.12.0"
+    latestReleaseDate: 2021-09-28
+
+-   releaseCycle: "3.11"
+    releaseDate: 2021-06-23
+    eol: 2022-06-30
+    lts: true
+    supportedJavaVersions: 8, 11
+    latest: "3.11.7"
+    latestReleaseDate: 2022-05-01
+
+-   releaseCycle: "3.10"
+    releaseDate: 2021-05-15
+    eol: 2021-05-15
+    lts: false
+    supportedJavaVersions: 8, 11
+    latest: "3.10.0"
+    latestReleaseDate: 2021-05-15
+
+-   releaseCycle: "3.9"
+    releaseDate: 2021-03-22
+    eol: 2021-03-22
+    lts: false
+    supportedJavaVersions: 8, 11
+    latest: "3.9.0"
+    latestReleaseDate: 2021-03-22
+
+-   releaseCycle: "3.8"
+    releaseDate: 2021-02-09
+    eol: 2021-02-09
+    lts: false
+    supportedJavaVersions: 8, 11
+    latest: "3.8.0"
+    latestReleaseDate: 2021-02-09
+
+-   releaseCycle: "3.7"
+    releaseDate: 2020-12-11
+    eol: 2021-12-31
+    lts: true
+    supportedJavaVersions: 8, 11
+    latest: "3.7.7"
+    latestReleaseDate: 2021-12-17
+
+-   releaseCycle: "3.6"
+    releaseDate: 2020-10-16
+    eol: 2020-10-16
+    lts: false
+    supportedJavaVersions: 8, 11
+    latest: "3.6.0"
+    latestReleaseDate: 2020-10-16
+
+-   releaseCycle: "3.5"
+    releaseDate: 2020-08-31
+    eol: 2020-08-31
+    lts: false
+    supportedJavaVersions: 8, 11
+    latest: "3.5.0"
+    latestReleaseDate: 2020-08-31
+
+-   releaseCycle: "3.4"
+    releaseDate: 2020-06-14
+    eol: 2021-06-30
+    lts: true
+    supportedJavaVersions: 8, 11
+    latest: "3.4.6"
+    latestReleaseDate: 2021-06-17
+
+-   releaseCycle: "3.3"
+    releaseDate: 2020-05-10
+    eol: 2020-05-10
+    lts: false
+    supportedJavaVersions: 8, 11
+    latest: "3.3.0"
+    latestReleaseDate: 2020-05-10
+
+-   releaseCycle: "3.2"
+    releaseDate: 2020-04-01
+    eol: 2020-04-01
+    lts: false
+    supportedJavaVersions: 8, 11
+    latest: "3.2.0"
+    latestReleaseDate: 2020-04-01
+
+-   releaseCycle: "3.1"
+    releaseDate: 2020-02-20
+    eol: 2020-02-20
+    lts: false
+    supportedJavaVersions: 8, 11
+    latest: "3.1.0"
+    latestReleaseDate: 2020-02-20
+
+-   releaseCycle: "3.0"
+    releaseDate: 2019-11-24
+    eol: 2020-02-20
+    lts: false
+    supportedJavaVersions: 8, 11
+    latest: "3.0.1"
+    latestReleaseDate: 2020-01-11
+
+-   releaseCycle: "2.25"
+    releaseDate: 2020-01-19
+    eol: 2022-01-31
+    lts: true
+    supportedJavaVersions: 8
+    latest: "2.25.4"
+    latestReleaseDate: 2021-05-24
+
+---
+
+> [Apache Camel](https://www.groovy-lang.org/) is an open source framework for message-oriented
```

Comment:
I think you got a copy/paste bug here , the link for camel doesnt point to it but groovy
correct link should be https://camel.apache.org/

Reply:
Fixed.